content/source/docs/cloud/api/run.html.md

```diff
@@ -51,6 +51,7 @@ Key path                    | Type   | Default | Description
 ----------------------------|--------|---------|------------
 `data.attributes.is-destroy` | bool | false | Specifies if this plan is a destroy plan, which will destroy all provisioned resources.
 `data.attributes.message` | string | "Queued manually via the Terraform Enterprise API" | Specifies the message to be associated with this run.
+`data.attributes.target-addrs` | array[string] | | Specifies a list of resource addresses to be passed to the `-target` flag.
```

Comment:
Do we add `| (nothing)` here to show it can be nil?

Reply:
It doesn't seem like we have a consistent way to indicate optional attributes! I originally left it blank since that's what we're doing for `workspace_id` below, but I don't have a strong opinion one way or the other so happy to update both of those to `(nothing)`